test.c

```diff
@@ -850,7 +850,7 @@ static void test_reply_reader(void) {
 
     test("Can parse RESP3 doubles in an array: ");
     reader = redisReaderCreate();
-    redisReaderFeed(reader, "*1\r\n,3.14159265358979323846\r\n",31);
+    redisReaderFeed(reader, "*1\r\n,3.14159265358979323846\r\n",30);
```

Comment:
```suggestion
    redisReaderFeed(reader, "*1\r\n,3.14159265358979323846\r\n",29);
```

I think maybe 29 because we don't want to include the `\0`? Seems like we're not including the null term elsewhere in `test.c`

Reply:
Oops, for some reason I checked the previous test case. But then put the wrong number anyways... Thanks!